package.json

```diff
@@ -130,6 +130,5 @@
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/people-power-solar/peoplepower-web/issues"
-  },
-  "homepage": "https://github.com/people-power-solar/peoplepower-web#readme"
```

Comment:
was causing local builds to fail, entered automatically when GitHub rebased our project from the original Blueprint fork.